app/javascript/components/miq-pagination.jsx

```diff
@@ -5,7 +5,7 @@ import { Pagination } from 'carbon-components-react';
 const getItemRangeText = (min, max, totalItems) => __(`${min}-${max} of ${totalItems} items`);
 
 // eslint-disable-next-line no-undef
-const getPageRangeText = (_current, total) => n__(`of ${total} page`, `of ${total} pages`, total);
+const getPageRangeText = (_current, total) => n__(__(`of ${total} page`), __(`of ${total} pages`), total);
```

Comment:
I think the right way is the other way around?

```suggestion
const getPageRangeText = (_current, total) => __(n__(`of ${total} page`), `of ${total} pages`, total));
```

Reply:
After talking with @MelsHyrule I thikn this is supposed to be:

```suggestion
const getPageRangeText = (_current, total) => sprintf(n__(`of %d page`, `of %d pages`, total), total);
```

The high level problem in the original code is that `${total}` is going to be interpolated first before going to translation.  `n__` will do the actual translation, but it is looking for something like `of 22 pages`, which doesn't exist in the catalog.  Instead, this suggestion looks for `of %d pages` in the catalog, then uses sprintf to smash in the values after wards.  Ruby had named replacement, but Javascript doesn't, which is why we have to use sprintf.  I'm thinking we should make a helper method to avoid sprintf, but for now that's what we have to do.